lib/phases/cold-launch.js

```diff
@@ -199,7 +199,11 @@ ColdLaunch.prototype.testRun = function() {
       }
 
       setTimeout(function() {
-        phase.device.log.memory(phase.appPid, entry.context);
+        phase
+          .triggerGC()
```

Comment:
I separated triggering GC from the logging of memory, otherwise in the reboot test we would try to GC twice and cause a race in the Marionette sessions. It was ugly. By separating it out, we can GC once and then log memory multiple times.